e2e-tests/cypress/integration/appeal-submission-to-horizon/appeal-submission-to-horizon.js

```diff
@@ -1,6 +1,51 @@
 import { Given, When, Then } from 'cypress-cucumber-preprocessor/steps';
 const { APPEAL_DOCUMENT } = require('../../../../packages/forms-web-app/src/lib/empty-appeal');
 
+const matchWhatWeCanFrom = (hardCodedExpectations) => {
```

Comment:
medium term this probably pops out into a helper or something, but it's likely good enough to work for the existing scenario + the new one we're adding in this branch..
if it stays in this branch it'll get tidied in the next 'horizon submission' story so up to you how hardcore you want to be here ;)